che-incubator/che-code
# Rebase against upstream VS Code to fix the auto-rebase job

In the PR, I've rebased Che-Code against upstream VS Code with resolving some conflicts in order to restore the automatic rebase GitHub Workflow.

closes https://github.com/eclipse/che/issues/21471

#### How to test
1. Run https://che-dogfooding.apps.che-dev.x6e0.p1.openshiftapps.com/#https://github.com/azatsarynnyy/java-spring-petclinic/tree/devfilev2.
2. Check that Che-Code is started.
